package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@transform-data/transform-react",
-  "version": "1.35.0",
+  "version": "1.35.1",
```

Comment:
Nit: Can you bump to 1.36.0?  If you're fixing a bug, bump the 0.0.x.  if you're adding a mutation/query bump 0.x.0.  If we're re-writing all the things, bump x.0.0

Reply:
Ahhh, good call, this was my bad. :)

Reply:
Ok! will do snd then merge